src/main/kotlin/com/cognifide/gradle/aem/base/api/AemConfig.kt

```diff
@@ -342,8 +349,18 @@ open class AemConfig(project: Project) : Serializable {
         instance(LocalInstance(httpUrl, user, password, type, debugPort))
     }
 
+    fun localAuthorInstance() {
```

Comment:
allows to define local instance with url in gradle.properties / also useful when forking